commcare/src/main/java/org/motechproject/commcare/events/FormActionEventHandler.java

```diff
@@ -42,7 +42,7 @@
 
     /**
      * Handles events, connected with sending Commcare forms. The event subject should have the following syntax:
-     * {@code EventSubjects.SEND_FORM.[form XMLNS].[config name]}
+     * {@code EventSubjects.SUBMIT_FORM.[form XMLNS].[config name]}
```

Comment:
This should not be changed, as it's the event subject, unless you want to rename the event subjects too.